sw/airborne/subsystems/radio_control/sbus_common.c

```diff
@@ -54,6 +54,124 @@
 #endif
 
 
+
+/*
+ * S.bus decoder matrix. (extracted from PX4)
```

Comment:
@kevindehecker could you test your SBUS2 radio with the `sbus2` branch of paparazzi repository (see https://github.com/paparazzi/paparazzi/commit/bba83260fffec4aa7c792ac48227d1366e047988) ? Thanks